webtau-docs/webtau/REST/HTTP-calls.md

```diff
@@ -0,0 +1,75 @@
+# Overloaded Calls
+
+`http.get|post|put|delete` methods have overloads to let you supply additional data:
+* Query parameters
+* Request header
+* Payload (*where applicable*) 
+* Validation block
+
+Overloads maintain relative order, but you can omit any data.
+
+`http.get` quick example:
+
+```tabs 
+Groovy: 
+:include-groovy: com/twosigma/webtau/http/HttpGroovyOverloadsTest.groovy {entry: "get no query return syntax example", bodyOnly: true, title: "without query"}
+
+Java: 
+:include-java: com/twosigma/webtau/http/HttpJavaOverloadsTest.java {entry: "getNoQueryReturnSyntaxExample", bodyOnly: true, title: "without query"}
+``` 
+
+# Implicit statusCode Check 
+
+If your code omits validation block, or if validation block doesn't assert `statusCode` an implicit status code validation
+be performed. 
```

Comment:
```suggestion
```

Reply:
I probably meant to say "will be performed". Your suggest to remove "be performed" part?

Reply:
This suggestion in isolation makes no sense.  I just moved the `be performed` part into the suggestion on the line above so deleted this line.

Reply:
It was done to make the previous suggestion less confusing :) but it seems to have had the opposite effect...